### PR TITLE
PBI 3632670 2019 .Net SDK Release 1

### PR DESCRIPTION
**Pull request by BrianBlum**
We explicitly ran ~\src\SDKs\Media\Management.Media\generate.ps1 to get --opt-in-extensible-enums.
The 'new' files in this PR are enum related files that were incorrectly deleted by https://github.com/Azure/azure-sdk-for-net/pull/5490.

### CODE REVIEW

**Comment by dsgouda** [comment]
@azuresdkci Retest this please

**Comment by dsgouda** [comment]
@azuresdkci Test this please

**Comment by BrianBlum** [comment]
This was supposed to be submitted to psSDKJson6. Closing this PR.

**Comment by dsgouda** [comment]
@BrianBlum reopening the PR. Our default branch has been switched to master now,

**Comment by BrianBlum** [comment]
@dsgouda, I didn't use the OpenAPI path to submit this code. Who should the reviewers be?

**Comment by dsgouda** [comment]
> 
> 
> @dsgouda, I didn't use the OpenAPI path to submit this code. Who should the reviewers be?

I will review this one. Will revert soon.

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part.
Acknowledging that apparent breaking changes are actually enums being reverted to extensible enums

File: src/SDKs/Media/Management.Media/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Media Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Media.")]
 
-[assembly: AssemblyVersion("2.0.1.0")]
-[assembly: AssemblyFileVersion("2.0.1.0")]
+[assembly: AssemblyVersion("2.0.2.0")]
```
[the remark this answers is not in the document]
Comment (BrianBlum) [reply src/SDKs/Media/Management.Media/Properties/AssemblyInfo.cs]: Thanks. Pushed new version of AssemblyInfo.cs.

**Review by dsgouda — APPROVED** [review]
LGTM will merge once CIs pass

**Comment by dsgouda** [comment]
Confirmed tests are fixed [here](https://azuresdkci.westus2.cloudapp.azure.com/view/NetCore-SDK/job/NetCore-SDK-Sign/502/)

**Pull request merged**